### ISSUE TITLE
Start ssh daemon last in the Windows templates to avoid premature shutdown

### ISSUE THREAD

**Issue by misheska** [misheska/basebox-packer#33]
Per Ross Smith,

The Windows templates should avoid starting the ssh daemon until after all the other "pre-install" scripts have run, to avoid premature shutdown.  In order to make the Windows 7 templates work with WinRM, they will be resetting/mucking around with the network interfaces during the postinstall phase - while Packer is pretty tolerant of the SSH connection being dropped, it is best just to not bother with this at all.

Plus, if one does a Windows update during the packer run, Packer definitely isn't as good at handling Windows reboots as it is Unix/Linux reboots.  Best to try to take care of the reboots before starting the ssh daemon.

Just start the sshd daemon last.

Rename to zz-start-sshd.cmd (or something like that), as the 00-run\* script runs scripts in alphabetical order.

**Comment by rasa** [comment]
Thanks for posting this. I've already done the work and will post the PR soon!

**Comment by misheska** [comment]
Thanks for providing good summaries of the issues!  Good to have them as issues so they can be tracked.
